scripts/AbinsModules/__init__.py

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
This looks like it has been copied from the main `mantid` file?

It should instead be a docstring about the `AbinsModules`. Can we get this updated?